scrapers/Lustery.yml

```diff
@@ -0,0 +1,19 @@
+name: Lustery
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://lustery.com
```

Comment:
should we limit the url to
```yaml
      - https://lustery.com/video-preview/
```
?

if we use a link like `https://lustery.com/videos#video-442` it wont work it needs `https://lustery.com/video-preview/442`

Reply:
Yes that is probably a good idea.

Reply:
Worth noting the canonical URL exists on the pages via

`<link rel="canonical" href="https://lustery.com/video-preview/442"/>`

Instead of filtering we also could grab canonical URL on the first scrape, and then once the URL is updated the user would be able to scrape properly. Might avoid user confusion figuring out why their link isn't working.

Reply:
If you use the `https://lustery.com/videos#video-442` link the canonical you get is `<link rel="canonical" href="https://lustery.com/videos"/>` and that's the page that stash downloads and why the scraper doesn't work with these links. So no it wouldn't work.